_includes/plugins/tinymcespellchecker_content_langs.md

```diff
@@ -0,0 +1,50 @@
+### `content_langs`
+{% include configuration/content_langs_base.md %}
+
+> **Note:** The `content_langs` option replaces the `spellchecker_select_languages` option
```

Comment:
This needs to be above all the content, not hidden in the middle. It's also missing a fullstop at the end of the sentence.